dist/index.js

```diff
@@ -28,31 +32,33 @@ exports['default'] = function () {
   var testAttempt = parsedOptions.testAttempt || 1;
   var logger = new _logger2['default'](parsedOptions.color);
 
-  function rerunFailedTests() {
+  function rerunFailedTests(status, output) {
```

Comment:
`output` is not needed

Reply:
Ok, let me take that up separately after I verify we don't miss anything because of this.